build/build.ps1

```diff
@@ -19,6 +19,7 @@ function Build-One {
         -v $Env:BUILD_VERBOSITY `
         /property:DefineConstants=$Env:ASSEMBLY_CONSTANTS `
         /property:Version=$Env:ASSEMBLY_VERSION `
+        /property:InformationalVersion=$Env:SEMVER_VESRION `
```

Comment:
```suggestion
        /property:InformationalVersion=$Env:SEMVER_VERSION `
```
Same in other files in this PR, and also I think in the similar changes in qsharp-compiler.